src/command_modules/azure-cli-vm/HISTORY.rst

```diff
@@ -4,6 +4,7 @@ Release History
 ===============
 2.2.3
 ++++++
+* Minor fixes.
```

Comment:
You only put this as a placeholder if there is nothing else customer-facing.

Reply:
Okay, makes sense.